tests/chaos/oadp/conftest.py

```diff
@@ -79,3 +90,11 @@ def drain_vm_source_node(rhel_vm_with_dv_running, oadp_backup_in_progress):
     with node_mgmt_console(node=vm_node, node_mgmt="drain"):
         wait_for_node_schedulable_status(node=vm_node, status=False)
         yield vm_node
+
+
+@pytest.fixture()
+def cordon_vm_source_node(rhel_vm_with_dv_running, oadp_backup_in_progress):
```

Comment:
not sure if we still do this for fixtures? `cordon` vs `cordoned` in naming:
`cordoned_vm_source_node`

Reply:
+1
should be `cordoned`